drivers/counter/Kconfig.nrfx

```diff
@@ -112,6 +112,19 @@ config COUNTER_RTC0_PRESCALER
 	help
 	  Frequency = 32768 / (prescaler+1).
 
+config COUNTER_RTC0_GUARD_PERIOD_MS
+	int "Guard period (milliseconds)"
+	default 10
+	range 1 10000
+	help
+	  Guard period should be set to the maximal latency that can occur when
+	  controlling the counter (e.g. for how long operation may be
+	  interrupted). Guard period limits maximal value that can be set to
+	  alarm to top value - guard period in ticks. Longer alarms will be
```

Comment:
Longer alarms .... will expire immediately?
Sounds weird...

Reply:
i elaborated a bit more. If it is still unclear please provide your suggestion.

Reply:
Now it is better.